proposals/exp-show-target.md

```diff
@@ -0,0 +1,210 @@
+- Enhancement Proposal PR: (leave this empty)
+- Contributors: dberenbaum, (add your Github handle)
+
+# Summary
+
+The `dvc exp show` command shows a comparison of the results of different
+experiments along with the parameters and metrics values for each experiment.
+The output currently includes all values that are defined in `params.yaml` or
+any other referenced parameters files in any DVC stage. Output also includes all
+metrics in any referenced metrics files in any DVC stage. This proposal allows
+users to see only parameters and metrics relevant to specified pipeline stages.
+
+See https://github.com/iterative/dvc/issues/5451 for more background.
```

Comment:
What's the difference between this proposal and https://github.com/iterative/dvc/issues/5451? Other than the nice format and summary, of course!

Reply:
Perhaps this was extracted from https://github.com/iterative/dvc/issues/5451#issuecomment-791205940 and that issue is focused on introducing the `--only-deps` flag?

Reply:
Yeah, the idea was to keep iterative/dvc#5451 focused on the short-term fix of adding an option to show only tracked parameters in `dvc metrics/params/exp show`. This proposal builds off the discussion there to introduce a potentially larger change specifically for `dvc exp show` (adding an argument for the stage of interest and only showing the parameters and metrics associated with that stage). In part, this is a trial run to see the pros and cons of moving larger feature changes into enhancement proposals, although this one is still pretty small by design.

Reply:
> adding an option to show only tracked parameters

Right that's what confused me, because "This proposal allows users to see only parameters and metrics relevant to specified pipeline stages," which seemed quite similar to iterative/dvc/pull/5550.

> potentially larger change specifically for dvc exp show (adding an argument for the stage of interest...

OK I see so `exp show clean` would display all the same experiments as rows, but only metrics and params for the `clean` stage as rightmost columns?

Reply:
> OK I see so `exp show clean` would display all the same experiments as rows, but only metrics and params for the `clean` stage as rightmost columns?

Correct. An "unresolved question" at the bottom of the doc is whether to include only the relevant experiments as rows.